src/plugins/inspector/public/views/requests/components/details/req_code_viewer.tsx

```diff
@@ -9,48 +9,35 @@
 // We want to allow both right-clicking to open in a new tab and clicking through
 // the "Open in Console" link. We could use `RedirectAppLinks` at the top level
 // but that inserts a div which messes up the layout of the inspector.
-/* eslint-disable @elastic/eui/href-or-on-click */
 
 import { EuiButtonEmpty, EuiCopy, EuiFlexGroup, EuiFlexItem, EuiSpacer } from '@elastic/eui';
 import type { ConnectionRequestParams } from '@elastic/transport';
 import { i18n } from '@kbn/i18n';
 import { XJsonLang } from '@kbn/monaco';
-import { compressToEncodedURIComponent } from 'lz-string';
-import React, { useCallback } from 'react';
-import { useKibana } from '@kbn/kibana-react-plugin/public';
+import React from 'react';
 import { CodeEditor } from '@kbn/code-editor';
-import { InspectorPluginStartDeps } from '../../../../plugin';
+import { RequestCodeViewerLinks } from './req_code_viewer_links';
 
 interface RequestCodeViewerProps {
   indexPattern?: string;
   requestParams?: ConnectionRequestParams;
   json: string;
+  includeOpenInAppLinks: boolean;
```

Comment:
Instead of using a flag, how about just passing in `actions: Array<{ name: string; action: ReactNode; }>`